### PR TITLE
Various license fixes to satisfy rat checks

### PR DESCRIPTION
**Pull request by matthiasblaesing**
Please see the individual commits for details/groups of changes.

In most cases the right license header was added.

### CODE REVIEW

**Comment by geertjanw** [comment]
Aside from the conflicting files, this looks good, will merge when conflicts are fixed.

File: websvc.saas.api/src/org/netbeans/modules/websvc/saas/model/wadl20061109.xsd [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+ or only the GPL Version 2, indicate your decision by adding
+ "[Contributor] elects to include this software in this distribution
+ under the [CDDL or GPL Version 2] license." If you do not indicate a
+ single choice of license, a recipient has the option to distribute
+ your version of this file under either the CDDL, the GPL Version 2 or
+ to extend the choice of license to its licensees as provided above.
+ However, if you add GPL Version 2 code and therefore, elected the GPL
+ Version 2 license, then the option applies only if the new code is
+ made subject to such option by the copyright holder.
+-->
```
Comment (junichi11) [inline websvc.saas.api/src/org/netbeans/modules/websvc/saas/model/wadl20061109.xsd]: Is this license header OK?

File: websvc.saas.api/src/org/netbeans/modules/websvc/saas/model/wadl200902.xsd [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+ or only the GPL Version 2, indicate your decision by adding
+ "[Contributor] elects to include this software in this distribution
+ under the [CDDL or GPL Version 2] license." If you do not indicate a
+ single choice of license, a recipient has the option to distribute
+ your version of this file under either the CDDL, the GPL Version 2 or
+ to extend the choice of license to its licensees as provided above.
+ However, if you add GPL Version 2 code and therefore, elected the GPL
+ Version 2 license, then the option applies only if the new code is
+ made subject to such option by the copyright holder.
+-->
```
Comment (junichi11) [inline websvc.saas.api/src/org/netbeans/modules/websvc/saas/model/wadl200902.xsd]: Is this license header OK?

**Comment by matthiasblaesing** [comment]
I pushed an updated changeset. Change from the first:
- the license header for the WADL xsds was extended to hold a clarification, that for redistribution the CDDL is choosen. This should enable us to redistribute the XSDs unter the https://www.apache.org/legal/resolved.html#category-b - last paragraph.
- the build failure was fixed. the eclipselink library definition covered more classes, than the replacement from maven central. The further necessary jars were added.

**Comment by geertjanw** [comment]
Great, merging.

**Comment by geertjanw** [comment]
Wait, before merging, there's still a conflict, can you resolve that?

**Pull request merged**